src/main/resources/db/migration/V1_176__update_referral_performance_report_first_attendance_srf.sql

```diff
@@ -0,0 +1,19 @@
+create or replace function performance_report_first_attendance(referral_id UUID)
```

Comment:
Just out of curiosity, how this function gets used in the main function?

Reply:
It's not called by the main report function @kth13 that's just used by the reader - it gets called at the later stage to find the appointment time for the processor (and added to the output)